clusters/l3-sqnc/capacity/nginx/source.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: bitnami
+  namespace: capacity
+spec:
+  interval: 10m
+  url: https://charts.bitnami.com/bitnami
```

Comment:
Move this into the Capacity level `source.yaml`

Reply:
This should be updated along with the values file. When you get a moment please take a look at the latest.